mw-webapp/src/pages/Page404/Page404.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from "src/pages/Page404/Page404.module.scss";
+
+const TEXT_ERROR = "404 NOT FOUND";
+
+export const Page404 = () => {
+  return (
+    <div className={styles.container}>
```

Comment:
I propose to use as different naming if possible. For example almost each component could contain container and it will be a little hard to find appropriate one when you are trying to debug etc. Much more easier to find  something more specific like errorPageContainer. It is just matter of code style, let's try this one